### ISSUE TITLE
Reviewing current button colour scheme

### ISSUE THREAD

**Issue by koderjoker** [internetarchive/iaux#33]
**Is your feature request related to a problem? Please describe.**
Most of the buttons in Internet Archive use Bootstrap's btn-primary, which fails WebAIM's colour contrast check
![Screen Shot 2019-03-25 at 10 01 40 PM](https://user-images.githubusercontent.com/30471843/54937185-f00b7380-4f49-11e9-8c77-6646ec64fe3e.png)
https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=428BCA
https://getbootstrap.com/docs/4.0/getting-started/accessibility/#color-contrast

**Describe the solution you'd like**
Rather than using bootstrap's provided button colours (which fail WebAIM's checks), we standardize our own set of colours to use throughout the website.

**Comment by pezvi** [comment]
@koderjoker Thank you, thank you, thank you for documenting this. We're also using Bootstrap's alerts, some of which (all of which?) also fail. Can you document that issue, too, please?

**Comment by koderjoker** [comment]
Sure, I'll add alerts to the above issue too and rename it.

**Comment by koderjoker** [comment]
@pezvi I can't find an instance where bootstrap's alert class is used at archive.org, could you please direct me to one?

**Comment by pezvi** [comment]
Yeah, now that I look, I'm not sure we're actually using bootstrap's alert class... maybe just their color schemes?

Some quick examples:
1. The help text on https://archive.org/create.php?lma=1 (hit the "?" icon to open the help box)
2. The borrow and join waitlist badging on item tiles https://archive.org/details/inlibrary?sort=-publicdate

**Comment by koderjoker** [comment]
@pezvi I'll add the above two examples to the issue, and keep adding on other examples with Bootstrap's colour scheme as I find them.

**Comment by iisa** [comment]
Though we are using standard Bootstrap element/status colors, we should look at this audit from an archive.org point of view.  I've updated the title to reflect that.
